spec/java/util/_interfaces.lsl

```diff
@@ -164,4 +174,15 @@ type ListIterator
     is java.util.ListIterator
```

Comment:
Maybe add "remove" fun ?

Reply:
The `Iterator` type already have that. What's the point of duplicating it there?